lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -29,13 +29,15 @@
                      * @returns {HtmlSnippet} - HTML list for course-wide discussion topics.
                      */
                     getCourseWideDiscussionsHtml: function(courseWideDiscussions) {
-                        var subCategoryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
+                        var entryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
```

Comment:
It may make sense to go with `subCategoryTemplate`, but it would be great if someone could help define category, subcategory, and entry.  I see from @muzaffaryousaf's link that my attempt to define these was wrong.  Are entries leaf subcategories?  Other?  I'm just not clear on what the names mean at this time.